app/helpers/breadcrumb_helper.rb

```diff
@@ -69,4 +69,9 @@ def training_provider_courses_breadcrumb
     path = training_provider_courses_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year, @training_provider.provider_code)
     training_providers_breadcrumb << ["#{@training_provider.provider_name}’s courses", path]
   end
+
+  def pe_allocations_breadcrumb
+    path = pe_allocations_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year)
+    provider_breadcrumb << ["Request PE courses for 2021/22", path]
```

Comment:
is it possible to not hard code the year and work it out based on the current recruitment cycle year?

Reply:
or is that out of scope?

Reply:
I was thinking the same thing but I think for the moment lets hard code it and refactor it later.

Reply:
I will add a card for this